.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.hs]
+indent_size = 2
+max_line_length = 80
```

Comment:
Do we want to restrict this to 80, or does 100 makes more sense?

Reply:
I would vote for 100.